### PR TITLE
feat(accordion): Add skeleton state

### PR DESCRIPTION
**Pull request by youda97**

### CODE REVIEW

File: src/accordion/accordion.component.ts [hunk 1]
```diff
@@ -8,6 +15,12 @@ import { Component } from "@angular/core";
...
 	`
 })
-export class Accordion {
+export class Accordion implements AfterContentInit {
+	@ContentChildren(AccordionItem) childs: QueryList<AccordionItem>;
 
+	@Input() skeleton = false;
+
+	ngAfterContentInit() {
+		this.childs.toArray().forEach(child => child.skeleton = this.skeleton);
```
Comment (cal-smith) [inline src/accordion/accordion.component.ts]: Same issue as breadcrumbs, we'll need something else to update the child state when the input changes. Also `children` not `childs`

File: src/accordion/accordion-item.component.ts [hunk 2]
```diff
@@ -21,17 +21,29 @@ import {
...
+				class="bx--accordion__title"
+				[ngClass]="{
+					'bx--skeleton__text': skeleton
+				}">
+				{{!skeleton ? title : null}}
+			</p>
 		</button>
 		<div [id]="id" class="bx--accordion__content">
-			<ng-content></ng-content>
+			<ng-content *ngIf="!skeleton; else skeletonTemp"></ng-content>
```
Comment (zvonimirfras) [inline src/accordion/accordion-item.component.ts]:
```suggestion
			<ng-content *ngIf="!skeleton; else skeletonTemplate"></ng-content>
```

File: src/accordion/accordion-item.component.ts [hunk 3]
```diff
@@ -21,17 +21,29 @@ import {
...
+			<ng-template #skeletonTemp>
```
Comment (zvonimirfras) [inline src/accordion/accordion-item.component.ts]:
```suggestion
			<ng-template #skeletonTemplate>
```

File: src/accordion/accordion-item.component.ts [hunk 4]
```diff
@@ -21,17 +21,29 @@ import {
 				viewBox="0 0 7 12">
           		<path fill-rule="nonzero" d="M5.569 5.994L0 .726.687 0l6.336 5.994-6.335 6.002L0 11.27z"/>
 			</svg>
-			 <p class="bx--accordion__title">{{title}}</p>
+			<p
+				class="bx--accordion__title"
+				[ngClass]="{
+					'bx--skeleton__text': skeleton
+				}">
+				{{!skeleton ? title : null}}
```
Comment (zvonimirfras) [inline src/accordion/accordion-item.component.ts]: Is it bad if we show the title regardless if it's available?

Comment (youda97) [reply src/accordion/accordion-item.component.ts]: If its available it will show in the loading state.. which we don't want happening

**Pull request merged**